test/unit/org/apache/cassandra/service/StartupChecksTest.java

```diff
@@ -103,6 +103,30 @@ public void compatibilityCheckIgnoresNonDbFiles() throws Exception
         startupChecks.verify();
     }
 
+    @Test
+    public void checkReadAheadKbSettingCheck() throws Exception
+    {
+        // This test just validates if the verify function
+        // doesn't throw any exceptions
+        startupChecks = startupChecks.withTest(StartupChecks.checkReadAheadKbSetting);
+        startupChecks.verify();
+    }
+
+    @Test
+    public void getReadAheadKBPath()
+    {
+        startupChecks = startupChecks.withTest(StartupChecks.checkReadAheadKbSetting);
+
+        String sdaDirectory = startupChecks.getReadAheadKBPath("/dev/sda12");
```

Comment:
use `StartupChecks.getReadAheadKBPath` to make explicit this is an static method.

Reply:
Changed. thanks.